drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -162,12 +162,21 @@ def self.static_options_for_type(pick_list_type, user:)
             { code: other_funder, label: other_funder }
           end
       when 'WORKFLOW_DEFINITION_TEMPLATES'
+        # used by the dropdown in the modal when you manually create an opportunity (will be removed)
         return [] unless Hmis::Ce.configuration.enabled?
 
         # TODO(#7502) - templates are shared across data sources
         Hmis::WorkflowDefinition::Template.all.map do |template|
           { code: template.id, label: template.name }
         end
+      when 'WORKFLOW_DEFINITION_TEMPLATE_IDENTIFIERS'
```

Comment:
Happy to discuss/change this around; for example, we could replace the original `WORKFLOW_DEFINITION_TEMPLATES` with the new one, instead of adding `WORKFLOW_DEFINITION_TEMPLATE_IDENTIFIERS`. (Related to considerations about when to remove [this](https://github.com/greenriver/hmis-warehouse/pull/5336))